<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptPopups</class>
 <widget class="QWidget" name="OptPopups">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptPopupsUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QCheckBox" name="ck_popupOn">
     <property name="text">
      <string>Enable popup notifications</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_type">
     <property name="title">
      <string>Notifications type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox1">
         <property name="title">
          <string>Popup notifications</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="ck_popupOnMessage">
            <property name="text">
             <string>On incoming message event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_showPopupMessage">
            <property name="text">
             <string>Show incoming message on popup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupOnHeadline">
            <property name="text">
             <string>On incoming headline event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupOnFile">
            <property name="text">
             <string>On incoming file event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupComposing">
            <property name="text">
             <string>When a contact begins typing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupOnOnline">
            <property name="text">
             <string>When a contact becomes available</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupOnOffline">
            <property name="text">
             <string>When a contact becomes unavailable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_popupOnStatus">
            <property name="text">
             <string>When a contact changes status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ck_everyMucMessage">
            <property name="text">
             <string>Enable popup notifications for every muc message</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Limitations</string>
           </property>
           <layout class="QGridLayout" name="_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Maximum JID length:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sb_jid">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Maximum text length:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="sb_text">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Maximum status length:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sb_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <number>-1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox2">
           <property name="title">
            <string>Popup location</string>
           </property>
           <layout class="QVBoxLayout" name="_2" stretch="0,0,0">
            <item>
             <widget class="QCheckBox" name="ck_topToBottom">
              <property name="text">
               <string>Show popups from top to bottom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ck_atLeft">
              <property name="text">
               <string>Place popups at left</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Avatar size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_avatar"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="gb_durations">
           <property name="title">
            <string>Durations, sec.</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QScrollArea" name="sa_durations">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="widgetResizable">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>538</width>
                 <height>81</height>
                </rect>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ck_popupOn</tabstop>
  <tabstop>ck_popupOnMessage</tabstop>
  <tabstop>ck_popupOnHeadline</tabstop>
  <tabstop>ck_popupOnFile</tabstop>
  <tabstop>ck_popupOnOnline</tabstop>
  <tabstop>ck_popupOnOffline</tabstop>
  <tabstop>ck_popupOnStatus</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ck_popupOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>tabWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_popupOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>gb_type</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
